robot_smach_states/src/robot_smach_states/utility.py

```diff
@@ -20,17 +20,18 @@ def __init__(self, robot=None):
         self.robot = robot
 
     def execute(self, userdata=None):
-        self.robot.lights.set_color(0, 0, 1)  # be sure lights are blue
-
-        self.robot.base.local_planner._action_client.cancel_all_goals()
-        self.robot.head.reset()
-        self.robot.leftArm.reset()
-        self.robot.leftArm.send_gripper_goal('close', 0.0)
-        self.robot.rightArm.reset()
-        self.robot.rightArm.send_gripper_goal('close', 0.0)
-        self.robot.ed.reset()
-        self.robot.torso.reset()
-        self.robot.hmi.restart_dragonfly()
+        # self.robot.lights.set_color(0, 0, 1)  # be sure lights are blue
```

Comment:
Are all these outcommented lines implemented in ```reset```s of the resp. parts?

Reply:
Yes, so every robot part that is initialized in the 'old' function now has a reset function with the same functionality.

Reply:
Aha, lights.py already had a [```reset```](https://github.com/tue-robotics/tue_robocup/blob/master/robot_skills/src/robot_skills/lights.py#L52) method, that's why I missed it here.